kong/db/schema/typedefs.lua

```diff
@@ -822,4 +823,9 @@ setmetatable(typedefs, {
 })
 
 
+typedefs.ttl = Schema.define {
+  type = "number",
+  between = { 0, DAO_MAX_TTL - 1 },
```

Comment:
Should be inclusive.

```suggestion
  between = { 0, DAO_MAX_TTL },
```